packages/lb-components/src/components/pointCloudView/hooks/usePointCloudViews.ts

```diff
@@ -761,7 +761,8 @@ export const usePointCloudViews = () => {
         synchronizeTopView(boxParams, newPolygon, topViewInstance, mainViewInstance);
       }
     }
-
+    pointCloudBoxList.push(boxParams);
```

Comment:
You can use  `newPointCloudList` from `const newPointCloudList = addPointCloudBox(boxParams);`. This way is error.